src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -144,4 +144,4 @@
     "tslint": "6.1.3",
     "typescript": "4.3.5"
   }
-}
+}
```

Comment:
can you remove this change @Sarthak0702 ?

Reply:
I did revert the changes. Looks like this was because of "Auto format on save" in vs code. Should I re-commit this (looks like a small change to me)?

Reply:
yup, could you please remove this and commit again?

Reply:
Thanks., have updated the code.